CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
-cmake_minimum_required(VERSION 3.13...3.16)
-if(${CMAKE_VERSION} VERSION_LESS 3.13)
+cmake_minimum_required(VERSION 3.17...3.20)
```

Comment:
The clang-tidy-review CI has only cmake 3.16 - so that workflow needs to be adapted.